.github/ISSUE_TEMPLATE/Epic.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Epic template
+about: Template for creating epics
+title: "[EPIC]"
+labels: Epic
+assignees: ''
+
+---
+
+<!--
+An Epic should define a higher-level package of work or project iteration that, once finished, makes sense to merge in core.
+-->
+## Description ##
+<!-- The what -->
+
+## Rationale ##
```

Comment:
Perhaps "Why" is better label, removing the comment too.

Reply:
Agreed, I was on the fence of using the same format as the PR template (What, Why)